app/board.js

```diff
@@ -32,8 +35,10 @@ const boardHandler = (req, res) => {
         throw new Error(boardRes.reason)
       }
 
-      const board = boardRes.value.data.payload.board_data
+      const board   = boardRes.value.data.payload.board_data
       const threads = board.threads
+      const count   = board.threads_count
+      const pages   = count <= limit ? [1] : Array.from({ length: Math.floor(count / limit) + 1 }, (v,i) => i + 1)
```

Comment:
Тут используется `Math.floor(..) + 1` - почему бы не использовать `Math.ceil(..)`? Тогда исчезнет прибавление магической единицы, а смысл останется тот же.

Да и необходимость в тернарке какая-то сомнительная. Я предлагаю сделать так
```suggestion
      const pages   = Array.from({ length: Math.ceil(count / limit) }, (v, i) => i)
```

Ещё короче и рабкотает. Безусловно, я, когда проверял, сделал ещё и соответствующие изменения в файле `views/pagination.pug`, чтобы там **не отнималась единица**.